rfc-117-containerised-runtime.md

```diff
@@ -0,0 +1,70 @@
+# RFC 117: Use a containerised runtime
```

Comment:
```suggestion
# RFC 117: Use a container runtime
```

While I don't have a strong understanding of "container" or "runtime", I think "container runtime" better reflects what this RFC is actually about.

I think most people would say a "containerised app" is an app deployed or running through a container. So a "containerised runtime" is a runtime deployed or running through a container, which I think isn't what this RFC is about.

Reply:
Thanks for looking at this @richardTowers , would it be possible to update other occurrences of "containerised runtime" within the text?

Reply:
Yeah - done in 898f8b35af (I haven't renamed the file though, because I'm concerned that will bamboozle GitHub - we can do that before merging if we remember)

Reply:
Ah, Erin's done the rename. Looks like GitHub is still okay 😸

Reply:
I've just renamed the file in 952877e and didn't think about breaking GitHub 😟